functions/menu.bash

```diff
@@ -222,6 +224,29 @@ show_main_menu() {
       "") return 0 ;;
       *) whiptail --msgbox "A not supported option was selected (probably a programming error):\\n  \"$choice2\"" 8 80 ;;
     esac
+
+  elif [[ "$choice" == "60"* ]]; then
+    choice2=$(whiptail --title "Welcome to the openHABian Configuration Tool $(get_git_revision)" --menu "Versioning" 16 116 9 --cancel-button Back --ok-button Execute \
+    "60 | Setup"      "Setup new or old Versioning using Github" \
```

Comment:
As said before it's probably please to explain your code first, then discuss repo structure, and halt further implementation until we have agreed upon a way forward.

One of the essential ideas of this feature is that an *operator* can *programmatically* up/downgrade by putting (rules,items, ...) code into the repo the
So it must work unattended in the first place.

Not sure about all the options you provide here to users. Experts will probably use git on the CLI anyway, and beginners may be overwhelmed by this.